docs/articles/azure/azs-how-create-key-vault-ps.md

```diff
@@ -0,0 +1,94 @@
+---
+title: How to create a Key Vault - PowerShell | UKCloud Ltd
+description: Provides help for creating a Key Vault on UKCloud for Microsoft Azure
+services: azure-stack
+author: Bailey Lawson
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: How to create a Key Vault - PowerShell

```

Comment:
Don't need to include the "How to" bit in the toc title. Should just be: Create a Key Vault - PowerShell

Reply:
Fixed